jetbrains-core/src/software/aws/toolkits/jetbrains/ui/RegionSelector.kt

```diff
@@ -44,9 +42,21 @@ class RegionSelector : ComboBox<AwsRegion>() {
             }
         }
 
-    private companion object {
-        val RENDERER = SimpleListCellRenderer.create<AwsRegion>("") {
+    companion object {
+        private val RENDERER = SimpleListCellRenderer.create<AwsRegion>("") {
             it.displayName
         }
+
+        fun Cell.regionSelector(prop: KMutableProperty0<AwsRegion>, serviceId: String? = null): CellBuilder<ComboBox<AwsRegion>> {
```

Comment:
clarify that the service id filters the region list?

Reply:
Will added